tmt.spec

```diff
@@ -79,7 +79,7 @@ Summary: Extra dependencies for the Test Management Tool
 Requires: tmt >= %{version}
 Requires: tmt-container >= %{version}
 Requires: tmt-testcloud >= %{version}
-Requires: vagrant python3-nitrate
+Requires: vagrant python3-nitrate Make
```

Comment:
I guess that should be `make`?

    # dnf install -y Make
    Last metadata expiration check: 1:36:28 ago on Wed Mar 11 09:24:36 2020.
    No match for argument: Make
    * Maybe you meant: make
    Error: Unable to find a match: Make

Reply:
yes, mistake during rebase
fixed e2e3953